source/precalculus/exercises/outcomes/EQ/EQ5/generator.sage

```diff
@@ -1,7 +1,7 @@
 load("../../../source/common/sagemath/library.sage")
 
 class Generator(BaseGenerator):
-  irrational_dict = TBIL.small_irrationals(rational_part=[-8..1,1..8],full_list=True)
+  irrational_dict = TBIL.small_irrationals(rational_part=[-8..-1,1..8],full_list=True)
```

Comment:
Missing negative allowed the rational part to be 0 in some cases, resulting in the sum of the roots being zero, and thus the coefficient of $x$ is zero.